packages/components/src/setupTests.ts

```diff
@@ -1,10 +1,15 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { type TestingLibraryMatchers } from '@testing-library/jest-dom/matchers';
 import '@testing-library/jest-dom/vitest';
-
 import { cleanup, configure } from '@testing-library/react';
 import { afterEach, beforeAll } from 'vitest';
 
+declare module 'vitest' {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    interface Assertion<T = any> extends jest.Matchers<void, T>, TestingLibraryMatchers<T, void> { }
+}
+
```

Comment:
out of scope for this PR

Reply:
I put this in so the pipeline does not fail:

The change is also in this separate PR:
https://github.com/Frontify/fondue/pull/1781